mautic/mautic
# Remove deprecated getUniqueIdentiferFields from FieldModel.

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 



## Description
Remove a long time deprecated method `getUniqueIdentiferFields` from `FieldModel`.







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Check tests are green.
3. Play with unique Contact fields.
